internal/kafka/command_topic_list.go

```diff
@@ -9,10 +9,18 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/output"
 )
 
-type topicOut struct {
+type topicNameOut struct {
 	Name string `human:"Name" serialized:"name"`
 }
 
+type topicOut struct {
+	Name              string `human:"Name" serialized:"name"`
+	Kind              string `human:"Kind" serialized:"kind"`
+	IsInternal        bool   `human:"Is Internal" serialized:"is_internal"`
+	ReplicationFactor int32  `human:"Replication Factor" serialized:"replication_factor"`
+	PartitionsCount   int32  `human:"Partitions Count" serialized:"partitions_count"`
```

Comment:
```suggestion
	PartitionCount   int32  `human:"Partition Count" serialized:"partition_count"`
```

Reply:
This aligns with the api response.. it's `topic.GetPartitionsCount()`

Reply:
Yeah... that's not proper grammar though :(

Reply:
(Maybe we should open an API ticket to fix this in the next major version of the API, too)

Reply:
^ Don't forget!